doc/source/devel/make_release.rst

```diff
@@ -130,11 +130,9 @@ Release checklist
 * Check on different platforms, particularly windows and PPC. Look at the
   `nipy buildbot`_ automated test runs for this.
 
-* Check the documentation doctests::
+* Check the documentation doctests (forcing Python 2)::
```

Comment:
I guess this isn't forcing Python 2, but forcing sphinx-build to run with the current `python`, whatever that is.